R/zchunk_L2322.Fert_CHINA.R

```diff
@@ -0,0 +1,499 @@
+             "L2322.SubsectorShrwtFllt_Fert_CHINA",
+             "L2322.SubsectorInterp_Fert_CHINA",
+             "L2322.SubsectorInterp_CHINAFert"))
+
+  } else if(command == driver.MAKE) {
+
+    all_data <- list(...)[[1]]
+
+    # Load required inputs
+    province_names_mappings <- get_data(all_data, "gcam-china/province_names_mappings")
+    calibrated_techs <- get_data(all_data, "energy/calibrated_techs")
+    A322.globaltech_coef <- get_data(all_data, "energy/A322.globaltech_coef")
+    L2322.Supplysector_Fert <- get_data(all_data, "L2322.Supplysector_Fert")
+    L2322.FinalEnergyKeyword_Fert <- get_data(all_data, "L2322.FinalEnergyKeyword_Fert")
+    L2322.SubsectorLogit_Fert <- get_data(all_data, "L2322.SubsectorLogit_Fert")
+    L2322.SubsectorShrwtFllt_Fert <- get_data(all_data, "L2322.SubsectorShrwtFllt_Fert")
+    L2322.SubsectorInterp_Fert <- get_data(all_data, "L2322.SubsectorInterp_Fert")
+    L2322.StubTech_Fert <- get_data(all_data, "L2322.StubTech_Fert")
+    L1322.IO_GJkg_province_Fert_F_Yh <- get_data(all_data, "L1322.IO_GJkg_province_Fert_F_Yh")
+    L1322.out_Mt_province_Fert_Yh <- get_data(all_data, "L1322.out_Mt_province_Fert_Yh")
+
+    # Silence package checks
+    regions <- supplysector <- subsector <- province <- value <- year <- subs.share.weights <-
+      technology <- share.weight.year <- minicam.energy.input <- coefficient <- market.name <-
+      sector <- fuel <- stub.technology <- grid.region <- region <- calOutputValue <-
+      subs.share.weight <- tech.share.weight <- logit.year.fillout <- logit.exponent <- logit.type <- NULL
+
+
+    # ===================================================
+    # In the old data system this chunk processed the user defined outputs,
+    # L2322.SubsectorInterpTo_Fert and L2322.SubsectorShrwt_Fert, that were produced by
+    # the upstream chunk, L2322.Fert. However as per discission with P. Kyle
+    # and K. Calvin these outputs were removed from the upstream chunk and therefore are no
+    # longer processed by this chunk.
+
+    # In the GCAM region CHINA N fertilizer is retained as a sector, as is the Imports subsector
+    # but the the fuel subsectors will be deleted and replaced with province subsectors. Subset the
+    # subsector logit exponents of fertilizer sector for the fuel subsectors to be removed in
+    # GCAM-CHINA.
+    L2322.SubsectorLogit_Fert %>%
+      filter(region == gcamchina.REGION, supplysector == "N fertilizer", subsector != "Imports") %>%
+      mutate(region = region) %>%
+      select(region, supplysector, subsector) ->
+      L2322.DeleteSubsector_CHINAFert
+
+    # Subset the supply sector keywords for fertilizer sector in the CHINA region.
+    L2322.FinalEnergyKeyword_Fert %>%
+      filter(region == gcamchina.REGION) %>%
+      mutate(final.energy = "none") ->
+      L2322.FinalEnergyKeyword_CHINAFert
+
+
+    # Since N fertilizer sectors are only created in provinces where the NAICS shipping information
```

Comment:
I think NAICS is only used for US, delete this comment since not relevant to China